build.cmd

```diff
@@ -55,6 +55,8 @@ set Platform=
 :: Log build command line
 set _buildproj=%~dp0build.proj
 set _buildlog=%~dp0msbuild.log
+set _binclashLoggerDll=%~dp0Tools\net45\Microsoft.DotNet.Build.Tasks.dll
```

Comment:
Yeah, build.cmd isn't using the core MSBuild so I'm assuming that the one in the user's path is the desktop one.  Once we move to using core-MSBuild on windows (like we do on unix) this needs to be updated.  It will throw on load if it isn't.